tests/test_openclhpp.cpp

```diff
@@ -4517,4 +4517,35 @@ void testgetObjectInfo() {
     TEST_ASSERT_EQUAL(type, CL_GL_OBJECT_BUFFER);
     TEST_ASSERT_EQUAL(bufobj, 0);
 }
+#if CL_HPP_TARGET_OPENCL_VERSION >= 120
+static cl_int clEnqueueMarkerWithWaitList_testenqueueMarkerWithWaitList(
+    cl_command_queue command_queue, cl_uint num_events_in_wait_list,
+    const cl_event *event_wait_list, cl_event *event, int num_calls) {
+    TEST_ASSERT_EQUAL_PTR(make_command_queue(0), command_queue);
+    TEST_ASSERT_EQUAL(1, num_events_in_wait_list);
+    TEST_ASSERT_NOT_NULL(event_wait_list);
+    TEST_ASSERT_EQUAL(0, num_calls);
+    if (event != nullptr) {
+            *event = make_event(1);
+    }
+    return CL_SUCCESS;
+}
+
+void testenqueueMarkerWithWaitList() {
+    cl_int ret = CL_DEVICE_NOT_FOUND;
+    cl::Event event;
+    cl::vector<cl::Event> events = {event};
```

Comment:
This is not valid "real-world" code - the set of events in the event wait list must be different than the event that is signaled.  Coudl you please make a specific event to wait upon instead?  This would also allow us to ensure that it was passed through correctly.

Something like:

```suggestion
    cl::Event event;
    cl::vector<cl::Event> events = {cl::Event(make_event(0))};
```